src/main/groovy/de/gliderpilot/gradle/semanticrelease/SemanticReleaseChangeLogService.groovy

```diff
@@ -237,6 +242,7 @@ class SemanticReleaseChangeLogService {
 
         Release release = repo.releases().create(tag)
         new Release.Smart(release).body(changeLog(commits(Version.valueOf(version.previousVersion)), version).toString())
+        releaseAsset(release.assets(), "$version.version")
```

Comment:
I'd like to send other kind of files too (.war, .exe), and a way to rename it. About of the file type the method URLConnection.guessContentTypeFromName() don't have all kind of files too. 

We can have a default implementation but I don't know the best way to do it.